src/App.spec.ts

```diff
@@ -437,20 +437,19 @@ describe('App', () => {
               respond: noop,
               ack: noop,
             },
-            // TODO: https://github.com/slackapi/bolt/issues/263
```

Comment:
nit: we also can verify if the condition works by having the following lines in the test suite.

```js
// add a new routing definition
app.view({callback_id: 'view_callback_id', type: 'view_closed'}, ({ }) => { viewFn(); })

// modify assert.equal(viewFn.callCount, 1); as below:
assert.equal(viewFn.callCount, 2);
```